fastapi/models/fastapi_endpoint.py

```diff
@@ -2,9 +2,10 @@
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL).
 
 import logging
+from collections.abc import Awaitable, Callable
```

Comment:
Awaitable ans Callable should be imported from typing

Reply:
Hi, they were deprecated since version 3.9 ( i read the information here: https://docs.python.org/3/library/typing.html)
![image](https://github.com/OCA/rest-framework/assets/8724793/4f489757-56c1-43e5-af8b-23b1773ac79d)